src/main/java/in/ashwanthkumar/gocd/artifacts/Janitor.java

```diff
@@ -92,11 +93,19 @@ public void run(String pathToConfiguration, Boolean dryRun) throws IOException {
 
         long bytesProcessed = performAction(whiteList, config.getArtifactStorage(), dryRun);
 
-        LOG.info("Total bytes (deleted / moved) processed so far - " + FileUtils.byteCountToDisplaySize(bytesProcessed));
+        LOG.info("Total bytes (deleted / moved) - " + FileUtils.byteCountToDisplaySize(bytesProcessed));
         LOG.info("Shutting down Janitor");
     }
 
-    /* default */ List<PipelineConfig> pipelinesNotInConfiguration(GoCD client, final JanitorConfiguration config) throws IOException {
+    private List<PipelineConfig> getPipelines() throws IOException {
+        LOG.info("Finding pipelines to process");
```

Comment:
If you run the Janitor without debug logging, there will be a pause here while the pipelines are loaded. I find it sensible to write an info line explaining what we are waiting for.